### PR TITLE
Test coordinates insert

### PR DESCRIPTION
**Pull request by kirs**
As discussed in https://github.com/WRSC/tracking/issues/10, let's cover the coordinates input format with tests.

### CODE REVIEW

**Comment by kirs** [comment]
@takluyver

File: MYR_rails/test/fixtures/coordinates.yml [hunk 1]
```diff
@@ -1,3 +1,15 @@
+<%
+module CoordinatesFixtureHelpers
+  def self.timeAddition(t1,t2)
+    # +1 is to stay in the same zone, may be this cannot work correctly if the server is set to an other zone than UTC + 3
```
Comment (takluyver) [inline MYR_rails/test/fixtures/coordinates.yml]: I don't follow this comment, or this function generally. It looks like we're converting a duration like 5 seconds to a timestamp (`t_ref + 5` below) and then back to a duration (`.to_i` here). I can see this is copy/pasted from somewhere else, but while we're copying it, let's take the time to make it simpler and clearer.

**Comment by kirs** [comment]
Thanks for comment @takluyver, I updated the helper.

File: MYR_rails/test/test_helper.rb [hunk 2]
```diff
@@ -4,6 +4,16 @@
 require "minitest/reporters"
 Minitest::Reporters.use!
 
+module CoordinatesFixtureHelper
+  extend self
+
+  def increment_time(base, delta)
```
Comment (kirs) [inline MYR_rails/test/test_helper.rb:10]: Here's the new test helper.

**Comment by takluyver** [comment]
Thanks!

**Pull request merged**